financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnectionsSheetCompose.kt

```diff
@@ -18,13 +19,18 @@ import com.stripe.android.financialconnections.launcher.FinancialConnectionsShee
 @Composable
```

Comment:
I think it's worth adding the extra documentation I added to the other Launchers:

> The GooglePayLauncher created is remembered across recompositions. Recomposition will always return the value produced by composition.

Reply:
https://github.com/stripe/stripe-android/pull/5275/commits/592c6a7c5cdfd410d090d380788d293c28bf0147